src/tools/check_brokenURL.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+
+import re
+import os
+import sys
+import getopt
+import requests
+import argparse
+
+def get_filepaths(directory):
+    file_paths = []  
+    for root, directories, files in os.walk(directory):
+        for filename in files:
+            filepath = os.path.join(root, filename)
+            file_paths.append(filepath)
+    return file_paths
+
+def checkBrokenLink(url):
```

Comment:
hi, @SpandanKumarSahu . there are some dummy links in our documentation. for example `dev/index.rst:429: [broken] https://github.com/mygithubaccount/ceph:
HTTP Error 404: Not Found`. We should be able to ignore those links.